.eslintrc

```diff
@@ -0,0 +1,96 @@
+{
+  "parser": "babel-eslint",
+  "env": {
+    "browser": true,
+    "node": true,
+    "es6": true
+  },
+  "ecmaFeatures": {
+    "templateStrings": true,
+  },
+  "rules": {
```

Comment:
In the interim, it would be nice (albeit counter to my point above) to pull in CoffeeLint to make that a requirement to pass before building our modules.